### PR TITLE
Image Component

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added Image Component

Live preview:https://framer-ui.netlify.app/docs/image.html

### CODE REVIEW

**Review by rahulk31 — APPROVED** [review]
Code for the components are well structured and seems easy to implement with less line of code. Classnames are descriptive enough. Good work. This looks good to merge. 👍

File: docs/image.html [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                </div>
+             </div>
+        </aside>
+    </nav>
+
+    <div class="container">
+        <div class="mt-3">
+            <h1 class="pb-1">Images</h1>
+            <p class="pb-1">Images can be responsive to fit the parent's width, and also can be customised to be round shaped</p>
+            <p class="pb-1">Yo can add <span class="text-red">.img-responsive</span> to make your image fit with container <span class="text-red">.img-responsive </span> uses <span class="text-red">width:100%, height:auto</span>.</p>
```
Comment (pradipgharge) [inline docs/image.html]: typo- YOU*

**Review by pradipgharge — APPROVED** [review]
No betterments, the code looks good. Good job on the UI.

File: docs/image.html [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+            <div class="card mt-1">
+               <div class="card-title">
+                <h3>Example of Responsive Image Banner</h3>
+               </div>
```
Comment (bhtibrewal) [inline docs/image.html:168]: Suggestion: i think you can remove this div and directly style the h3

File: docs/image.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+                            <circle fill="black" cx="12.5" cy="10" r="0"></circle>
+                        </mask>
+                        <rect fill="hsl(139, calc(var(--saturation-factor, 1) * 47.3%), 43.9%)" width="25" height="15" mask="url(#2b3fa5a7-0ec2-4469-b164-f8d5007235b6)"></rect>
+                    </svg>
+                    <rect x="22" y="22" width="10" height="10" fill="transparent" aria-hidden="true" class="pointerEvents-2zdfdO"></rect>
+                </svg>
+                <div class="flex flex-column">
+                    <span class="font-white">Made 😎 by Mehul </span>
+                    
+                </div>
```
Comment (bhtibrewal) [inline docs/image.html:155]: this div seems extra try to remove it

File: docs/image.html [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+                            <polygon points="-2.16506,-2.5 2.16506,0 -2.16506,2.5" fill="black" transform="scale(0) translate(13.125 10)" style="transform-origin: 13.125px 10px;">
+                            </polygon>
+                            <circle fill="black" cx="12.5" cy="10" r="0"></circle>
+                        </mask>
+                        <rect fill="hsl(139, calc(var(--saturation-factor, 1) * 47.3%), 43.9%)" width="25" height="15" mask="url(#2b3fa5a7-0ec2-4469-b164-f8d5007235b6)"></rect>
+                    </svg>
+                    <rect x="22" y="22" width="10" height="10" fill="transparent" aria-hidden="true" class="pointerEvents-2zdfdO"></rect>
+                </svg>
+                <div class="flex flex-column">
+                    <span class="font-white">Made 😎 by Mehul </span>
```
Comment (bhtibrewal) [inline docs/image.html:153]: Suggestion: move this to a footer section

**Review by bhtibrewal — APPROVED** [review]
I have written a few suggestions, look into them, else everything looks good... nice clean code 
good to merge

**Review by j24m — COMMENTED** [review]
Make the changes suggested by other and then you are good to go .

**Pull request merged**